mk/spksrc.service.mk

```diff
@@ -7,7 +7,7 @@
 #   conf/SPK_NAME.sc       if SERVICE_PORT and DSM<7
 #   conf/resource          if SERVICE_CERT or DSM7
 #   app/SPK_NAME.sc        if SERVICE_PORT and DSM7
-#   app/config             if DSM_UI_DIR
+#   app/config             if DSM_UI_DIR (may be overwritten by DSM_UI_CONFIG)
```

Comment:
@ymartin59, @publicarray some time ago we added the generation of `app/config` files.
For this package we do not want the generated `app/config` for the service but one for the included phpMemcachedAdmin web interface.

The app/config file is used with DSM < 7 only, as for DSM 7 the web UI is defined in the resources file and app/config seems to be ignored.

So a lot of packages still have a `src/app/config` file in the spk folder, and we will have to validate, whether those match the generated one. Matching config files could be removed and for different files we have to apply `DSM_UI_CONFIG` too.